JuliaInterface/README.md

```diff
@@ -50,8 +50,7 @@ To do so, run configure as follows:
 ## Usage
 
 Load the JuliaInterface package via
-```
-LoadPackage( "JuliaInterface" );
-```
 
+    LoadPackage( "JuliaInterface" );
```

Comment:
I like the GitHub style here more than plain markdown. However, it is probably better this way.